spec/models/user_spec.rb

```diff
@@ -0,0 +1,107 @@
+require "rails_helper"
+
+RSpec.describe User, type: :model do
+  let(:user) { FactoryBot.create :user }
+
+  describe "validates" do
+    it do
+      is_expected.to validate_presence_of(:name)
```

Comment:
Nên tách ra
1 it chỉ chứa 1 expectation. lợi ích dễ đọc hơn. lúc failed dễ fix hơn
```suggestion
    it { is_expected.to validate_presence_of(:name) }
```